### PR TITLE
Modification to 1.1.1

### PR DESCRIPTION
**Pull request by Footrotflat**
Fixed - parameter createAction - PHP7 compatibility

### CODE REVIEW

File: Classes/Controller/EditController.php [hunk 1]
```diff
@@ -12,6 +12,18 @@
 class EditController extends EditControllerFemanager
 {
 
+        /**
+     * Initialize update action for setting the right custom data type for the user.
+     */
+    public function initializeCreateAction() {
```
Comment (Keinbockwurst) [inline Classes/Controller/EditController.php]: I think this is wrong.. shouldnt it be initializeUpdateAction() ?

**Review by Keinbockwurst — CHANGES_REQUESTED** [review]

File: Classes/Controller/EditController.php (hunk 1 above)
Comment (Keinbockwurst) [reply Classes/Controller/EditController.php]: I tested this and yup, its wrong.

**Comment by Footrotflat** [comment]
Yes you are right

**Comment by werkraum-admin** [comment]
You can avoid the xClass by setting the ObjectType via the PropertyMapper

`/** @var \TYPO3\CMS\Extbase\Mvc\Controller\Argument $user */
            $user = $this->arguments['user'];
            $user->getPropertyMappingConfiguration()->setTargetTypeForSubProperty('user',
                \In2code\Femanagerextended\Domain\Model\User::class);`

**Pull request closed without merging**